development/patch_schema.py

```
@@ -0,0 +1,46 @@
+frappe.connect()
+
+FIXES = [
+    # (table, column, definition)
+    ("tabFOSS Event CFP Review", "parent",      "varchar(140) DEFAULT NULL"),
+    ("tabFOSS Event CFP Review", "parentfield", "varchar(140) DEFAULT NULL"),
+    ("tabFOSS Event CFP Review", "parenttype",  "varchar(255) DEFAULT NULL"),
+]
+
+for table, column, defn in FIXES:
+    existing = frappe.db.sql(
+        f"SELECT COLUMN_NAME FROM INFORMATION_SCHEMA.COLUMNS "  # noqa: S608
```

Ruff (0.15.12)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)